app/src/main/res/values/strings.xml

```diff
@@ -203,7 +203,7 @@
     <string name="gadgetbridge_running">Gadgetbridge running</string>
     <string name="installing_binary_d_d">Installing binary %1$d/%2$d</string>
     <string name="installation_failed_">Installation failed</string>
-    <string name="installation_successful">Installation successful</string>
+    <string name="installation_successful">Installed</string>
```

Comment:
What is wrong about successful here?

Reply:
"Successful" is always redundant. It also implies something usually goes wrong, or has gone wrong already, otherwise it would be omitted.